### PR TITLE
Initial template for SIG Cloud Provider 2020 Annual Report

### LINKED ISSUE #5500: 2021 Annual Report: SIG Cloud Provider
Chairs: @andrewsykim @cheftako 
Liasion: @nikhita

- [x] Consult SIG to complete draft of report
- [ ] If needed, consult with Steering Committee liasion on any private issues or concerns that arrise while completing report
- [x] Submit PR to git.k8s.io/community repo with copy of report, assigning your SC liasion for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, SC will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

Key dates:
- Initial PR to communtiy repo should be opened by March 1st, 2021 (DONE)
- PR should be reviewed and merged by ~April 1st, 2021~ April 8th, 2021

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liasion directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig cloud-provider
/committee steering

### PR DESCRIPTION
**Pull request by cheftako**
Adding initial template for SIG Cloud Provider 2020 Annual Report.
Part of issues/5500.
This is due by March 1st and will be shown to the SIG and Slack.

**Which issue(s) this PR fixes**:
Fixes #5500

### CODE REVIEW

File: sig-cloud-provider/2020-annual-report.md
Comment (jiahuif) [inline sig-cloud-provider/2020-annual-report.md:62]: https://github.com/kubernetes/enhancements/issues/2436 to alpha
(add cred. provider too?)

File: sig-cloud-provider/2020-annual-report.md
Comment (jpbetz) [inline sig-cloud-provider/2020-annual-report.md:38]: I feel like there are two answer to this:

- For changes to cloud provider infrastructure where all approvers can approve, I think things are going reasonably well.
- For changes to a specific cloud provider implementation, where an approval is required from someone outside the company, I think there is a bigger problem, particularly for VMWare and Google since our SIG Chairs (our most active approvers) are from those respective companies.  I feel like we need to work to work toward having more approvers so (e.g. get to the point where the defacto bus factor is at least 2 for any change). I don't think this needs to imply that the immediate goal should be to add more top level approvers-- getting more sub-tree approvers and expanding the ownership of existing sub-tree approvers seems like a good path forward.

Curious what others think.

**Comment by nikhita** [comment]
@cheftako :wave: The annual reports need to be merged by April 1st (next Thursday). If you could update the PR to add more details and share it with the cloud-provider mailing list by Monday/Tuesday next week, that'd be great!

**Comment by cheftako** [comment]
> Walter Fender  The annual reports need to be merged by ~April 1st (next Thursday)~ April 8th. If you could update the PR to add more details and share it with the cloud-provider mailing list by next week, that'd be great!

Sent

File: sig-cloud-provider/2020-annual-report.md
Comment (parispittman) [inline sig-cloud-provider/2020-annual-report.md:40]: thanks for this explanation

File: sig-cloud-provider/2020-annual-report.md
Comment (parispittman) [inline sig-cloud-provider/2020-annual-report.md:60]: are all of these long tail or things you'd like to highlight as awesome work?

File: sig-cloud-provider/2020-annual-report.md
Comment (parispittman) [inline sig-cloud-provider/2020-annual-report.md:83]: what do you attribute this to?

**Review by parispittman — CHANGES_REQUESTED** [review]

File: sig-cloud-provider/2020-annual-report.md
Comment (nckturner) [inline sig-cloud-provider/2020-annual-report.md:28]: Where are subproject owners listed?

File: sig-cloud-provider/2020-annual-report.md
Comment (nckturner) [reply sig-cloud-provider/2020-annual-report.md:22]: @cheftako I think the extraction migration subproject owners needs an update.

Comment (cheftako) [reply sig-cloud-provider/2020-annual-report.md:22]: Sadly I believe that subproject owner is currently me. However I firmly believe all WG/SubProjects should have at least two owners. Care to join me?

File: sig-cloud-provider/2020-annual-report.md
Comment (cheftako) [reply sig-cloud-provider/2020-annual-report.md:60]: highlighted as awesome work

File: sig-cloud-provider/2020-annual-report.md
Comment (cheftako) [reply sig-cloud-provider/2020-annual-report.md:83]: Still trying to build up a team of multiple contributors for every cloud provider to help triage and handle these. We are slowly growing this and its getting better but more so with some cloud providers than others.

File: sig-cloud-provider/2020-annual-report.md
Comment (cheftako) [reply sig-cloud-provider/2020-annual-report.md:28]: I believe they are listed in https://github.com/kubernetes/community/blob/master/sigs.yaml. These are mostly then redirects to other owners files. There is also https://github.com/kubernetes/community/blob/master/OWNERS_ALIASES but non of our sub-projects are listed there.

File: sig-cloud-provider/2020-annual-report.md
Comment (nckturner) [reply sig-cloud-provider/2020-annual-report.md:22]: Happy to!

**Comment by cheftako** [comment]
@nikhita Can we remove the hold?

**Comment by nikhita** [comment]
> @nikhita Can we remove the hold?

Yes, looks good! Let's :shipit: !!

/lgtm
/hold cancel

**Pull request merged**
